telco-core/configuration/reference-crs/optional/other/mount_namespace_config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-kubens-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - enabled: true
+        name: kubens.service
```

Comment:
Aren't you missing the service definition? Or is it included by default now?

Reply:
@lack Would you know this information?

Reply:
@sebrandon1 Well compare the references RAN RDS yaml https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.16/ztp/source-crs/extra-manifest/01-container-mount-ns-and-kubelet-conf-worker.yaml with what is in this PR here..

Reply:
Instead of choosing one winner, I just added the other two YAMLs into the same directory.

Reply:
Oh, I see where the confusion comes from. kubens is a different method from what you mentioned in https://github.com/openshift-kni/telco-reference/pull/15#issuecomment-2234341208

I think you need to update the PR to carefully explain what it is adding and why for every single added manifest.

Are the kubens and 01-container-mount-ns-and-kubelet-conf pieces additive or conflicting? Or is one a newer version of the other?

Reply:
@lack can you review this for correctness against the current OpenShift supported method. Thanks!

Reply:
@lack Can you please bring some light to what should be done here?

Reply:
So for Core we should include only the kubens.service version because there is no history of use of the old version of this. @sebrandon1 can you update this PR to use only the mount_namespace_config.yaml version, but also split it into two different files, one for master and one for worker. Thanks!

Reply:
Okay sounds good.  This has been updated in: [9287cf4](https://github.com/openshift-kni/telco-reference/pull/15/commits/9287cf40636fc8bc799f47608c4eebdc54544b90)